FinniversKit/Sources/Components/ObjectPagePriceView/Subviews/PriceLinkButton.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+protocol PriceLinkButtonViewDelegate: AnyObject {
+    func priceLinkButton(withIdentifier identifier: String?, wasTappedWithUrl url: URL)
+}
+
+class PriceLinkButtonView: UIView {
+
+    // MARK: - Internal properties
+
+    weak var delegate: PriceLinkButtonViewDelegate?
+
+    // MARK: - Private properties
+
+    private let viewModel: PriceLinkButtonViewModel
+    private let linkButtonStyle = Button.Style.link.overrideStyle(smallFont: .body)
+    private lazy var fillerView = UIView(withAutoLayout: true)
+    private lazy var externalImage = UIImage(named: .webview).withRenderingMode(.alwaysTemplate)
+
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView(withAutoLayout: true)
+        stackView.axis = .vertical
+        stackView.alignment = .top
+        return stackView
+    }()
+
+    private lazy var buttonStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [linkButton, fillerView, externalImageView])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.alignment = .center
+        return stackView
+    }()
+
+    private lazy var linkButton: Button = {
+        let button = Button(style: linkButtonStyle, size: .small, withAutoLayout: true)
+        button.addTarget(self, action: #selector(handleTap), for: .touchUpInside)
+        button.contentHorizontalAlignment = .leading
+        return button
+    }()
+
+    private lazy var externalImageView: UIImageView = {
+        let imageView = UIImageView(withAutoLayout: true)
+        imageView.image = externalImage
+        imageView.tintColor = .externalIconColor
+        return imageView
+    }()
+
+    private lazy var headingLabel: Label = {
+        let label = Label(style: .body, withAutoLayout: true)
+        label.textColor = .textPrimary
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private lazy var subheadingLabel: Label = {
+        let label = Label(style: .bodyStrong, withAutoLayout: true)
+        label.textColor = .textPrimary
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private lazy var subtitleLabel: Label = {
+        let label = Label(style: .detail, withAutoLayout: true)
+        label.textColor = .stone
```

Comment:
Shouldn't it be dark mode compatible? 🌔

Reply:
`.stone` seems to work fine in darkmode as well, but just to be sure I'll ask the designers

![Screenshot 2020-08-26 at 13 26 24](https://user-images.githubusercontent.com/1901556/91298137-c8a08200-e79f-11ea-979c-5975ac75dabd.png)

Reply:
I see, was wondering if we should use `.textSecondary` which has a dark mode version of `.stone` 😄

Reply:
Nice! Using that one instead😄👍